packages/nextra-theme-blog/package.json

```diff
@@ -32,13 +32,15 @@
   "scripts": {
     "build": "tsup",
     "build:tailwind": "cross-env NODE_ENV=production pnpm postcss src/styles.css -o style.css --verbose",
+    "build:all": "pnpm build && pnpm run build:tailwind",
     "types": "tsup --dts-only",
     "types:check": "tsc --noEmit",
-    "dev": "concurrently \"pnpm dev:layout\" \"pnpm dev:tailwind\"",
+    "dev": "concurrently \"pnpm dev:layout\" \"pnpm dev:tailwind:workspace\"",
     "dev:layout": "tsup --watch",
+    "dev:tailwind:workspace": "cross-env TAILWIND_MODE=watch pnpm postcss src/styles.css -o style_node_modules.css --watch",
```

Comment:
Do you still need it even with `"injected": true`?

Reply:
forget to remove this